src/GettingStarted.md

```diff
@@ -38,31 +25,36 @@ Give your repo a name and click on "Create repository":
 <img src="img/my-3pi.png" alt="new repo"/>
 
 ## Clone your repository
-On the command line on your host machine, clone the repo.
-For an SSH-based setup, run:
-
-```bash
-$ git clone git@github.com/<username>/<reponame>.git
-```
-
-Or, for a setup that uses GitHub CLI, run:
+On the command line on your host machine, change directory to the location where you would like to check out your repository. Let us assume that you named your repo `my-3pi`. Check it out using the following command (where `<username>` must be substituted with your GitHub username):
 
 ```bash
 $ gh repo clone <username>/<reponame>
 ```
 
+This will create a directory called `my-3pi` in the current working directory.
+
 The template includes [raspberrypi/pico-sdk](https://github.com/raspberrypi/pico-sdk) as a submodule, which itself also has a lot of submodules. We recommend against using the `--recursive` flag because we do not need to recursively clone the submodules inside of `pico-sdk`. Instead, change directory into the root of your clone and run:
 
 ```bash
 $ git submodule update --init
 ```
 
-If  `pico-sdk` was checked out correctly, `git submodule` will show the hash _without_ a `-` preceding it,
+If  `pico-sdk` was checked out correctly running `git submodule` in the root of the repository will show the hash _without_ a `-` preceding it,
 e.g.: `6a7db34ff63345a7badec79ebea3aaef1712f374 pico-sdk (1.5.1)`.
 
+> **_Note for existing GitHub users_**
+>
+> If you are an existing GitHub user and have already set up a public/private key pair (or have done so by selecting `SSH` as the protocol when running `gh auth login`), you can also clone the repo as follows:
+> ```
+> $ git clone git@github.com/<username>/<reponame>.git
+> ```
 
 ## Configure Nix
 
+> **_Note for VM users_**
+>
+> If you are using the [VM image](https://vm.lf-lang.org/), you can skip this step. You will never have to invoke `nix` and can ignore any reminders about doing this.
```

Comment:
@perry0513 and @victoria-tuck, please confirm that this is indeed the case. If there is still any dependency missing, I will add it.

Reply:
I'm able to compile the examples after pulling in the submodules of `pico-sdk`. Will add those to the VM (hope it won't blow up too much), and publish it.